roles/openshift_node/tasks/main.yml

```diff
@@ -59,6 +68,7 @@
     dest: /etc/sysconfig/{{ openshift.common.service_type }}-node
     regexp: "{{ item.regex }}"
     line: "{{ item.line }}"
+    create: yes
```

Comment:
Is this file no longer provided by the packaging? I'd want to avoid creating it in ansible if at all possible.